sdk/ml/tests.yml

```diff
@@ -6,10 +6,9 @@ stages:
       ServiceDirectory: ml
       MatrixReplace:
         - TestSamples=.*/true
+      CloudConfig:
+        Public:
+          SubscriptionConfiguration: $(python-ml-subscription-scope-configuration)
       EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
```

Comment:
FYI you may need `AZURE_CLIENT_ID: $(ML_CLIENT_ID)` and equivalents, unless your tests handle these namespaced env variable lookups already.

Reply:
our tests do look up the environment variables by namespace (i.e. they will look for `ML_...`). I don't need to include these, right?

Reply:
Correct, you don't need to include them.